kinesis/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/sinks/KinesisSink.scala

```diff
@@ -138,36 +136,17 @@ object KinesisSink {
                 exception: AmazonClientException,
                 retriesAttempted: Int
               ): Boolean =
-                retriesAttempted < 10 &&
-                  (exception match {
-                    case _: ProvisionedThroughputExceededException => false
-                    case _                                         => true
-                  })
+                false
```

Comment:
Is the assumption here that we don't need to check if this should be retried because we would only deploy this asset when Kinesis is down and so we know that there is no point in retrying?

Reply:
Yes, exactly as you said. Don't loose time on retries, when we know kinesis is down.